js/data.json

```diff
@@ -430,6 +430,30 @@ openings = [
 		"link" : "http://67.159.62.2/anime_ost/steins-gate-op-single-hacking-to-the-gate/jntezehukc/01%20Hacking%20to%20the%20Gate.mp3",
 		"name" : "Steins;Gate OP Single - Hacking to the Gate",
 		"img" : "https://images2.alphacoders.com/224/thumb-1920-224760.jpg"
+	},
+    
+	{
+		"link" : "http://67.159.62.2/anime_ost/gintama-op2-single-tooi-nioi/dgrvnaggyd/01%20-%20Tooi%20Nioi.mp3",
+		"name" : "Gintama OP2 - Tooi Nioi",
+		"img"  : "https://animejanaika.files.wordpress.com/2011/04/horriblesubs-sket-dance-01-720p-mkv_snapshot_23-45_2011-04-11_01-59-08.jpg"
+	},
+
+	{
+		"link" : "http://67.159.62.2/anime_ost/gintama-op3-single-gin-iro-no-sora/cjbijtbyxz/01%20-%20Gin%20iro%20no%20Sora.mp3",
+		"name" : "Gintama OP3 - Gin iro no Sora",
+		"img"  : "https://myanimelist.cdn-dena.com/s/common/uploaded_files/1449204090-b9875961dca681114b8c47224457c13b.jpeg"
+	},
+
+	{
+		"link" : "http://67.159.62.2/anime_ost/gintama-op4-single-kasanaru-kage/mjydqwsuom/01%20-%20Kasanaru%20Kage.mp3",
+		"name" : "Gintama OP4 - Kasanaru Kage",
+		"img"  : "http://image.tmdb.org/t/p/w780/apO3905dT9eyWnrDtT3BGASziP0.jpg",
```

Comment:
Image have low resolution, may get pixelated.

Reply:
Yea, try to keep images >720p